### PR TITLE
Final skeleton revision

### PR DESCRIPTION
**Pull request by TryExceptElse**
Incorporated commenting changes from backend-3 revision, and made changes following recommendations made during 6-2 presentations

### CODE REVIEW

File: src/main/java/io/github/tryexceptelse/jdex/JDex.java [hunk 1]
```diff
@@ -1,93 +1,225 @@
...
+      ContactEntry emailAddressTest = new EmailAddress();
+      ContactEntry firstNameTest = new FirstName();
+      ContactEntry lastNameTest = new LastName();
+      ContactEntry phoneNumberTest = new PhoneNumber();
+      ContactEntry streetAddressTest = new StreetAddress();
+      //
+      // Frontend Objects:
+      //
+      AppWindow appWindowTest = new AppWindow();
+      Gui guiTest = new Gui(JDexTest);
```
Comment (Entrer1) [inline src/main/java/io/github/tryexceptelse/jdex/JDex.java]: JDexTest isn't a class. J should be lowercase. This confused me for a while.

File: src/main/java/io/github/tryexceptelse/jdex/JDex.java [hunk 2]
```diff
@@ -104,38 +237,59 @@ public void setUi(Ui ui) {
...
    * builds user interface based on information in settings
+   *
    * @return Ui: User Interface object
    */
-  private Ui buildUi(){
-    return new Gui(this, runtimeArgs); //placeholder.
-      // Should first check settings for type of GUI
+  private Ui buildUi() {
+    return new Gui(this); //placeholder.
+    // Should first check settings for type of GUI
```
Comment (Entrer1) [inline src/main/java/io/github/tryexceptelse/jdex/JDex.java]: Didn't we get rid of Settings? Is just just holdover?

File: src/main/java/io/github/tryexceptelse/jdex/JDex.java [hunk 3]
```diff
@@ -104,38 +237,59 @@ public void setUi(Ui ui) {
...
   }
 
-}
+  /**
+   * creates contacts File object to be stored and
+   * used for saving and loading data
+   * @return File to save to and load from.
+     */
+  private File buildContactsFile() {
+      // File does not accept a default constructor call.
```
Comment (Entrer1) [inline src/main/java/io/github/tryexceptelse/jdex/JDex.java:288]: Why doesn't File accept default constructor?

File: src/main/java/io/github/tryexceptelse/jdex/JDex.java (hunk 2 above)
Comment (TryExceptElse) [inline src/main/java/io/github/tryexceptelse/jdex/JDex.java]: Yeah, this is just old commenting.

File: src/main/java/io/github/tryexceptelse/jdex/JDex.java (hunk 3 above)
Comment (TryExceptElse) [inline src/main/java/io/github/tryexceptelse/jdex/JDex.java:288]: its methods presume it represents an actual file location that can be accessed and read from/written to

Comment (TryExceptElse) [inline src/main/java/io/github/tryexceptelse/jdex/JDex.java]: its methods presume it represents an actual file location that can be accessed and read from/written to

File: src/main/java/io/github/tryexceptelse/jdex/JDex.java (hunk 1 above)
Comment (TryExceptElse) [inline src/main/java/io/github/tryexceptelse/jdex/JDex.java]: My bad, I had it named 'testJDex' before. Didn't change the case when I renamed it.

File: src/main/java/io/github/tryexceptelse/jdex/fe/gui/MainCont.java [hunk 4]
```diff
@@ -1,15 +1,53 @@
 package io.github.tryexceptelse.jdex.fe.gui;
```
Comment (Entrer1) [inline src/main/java/io/github/tryexceptelse/jdex/fe/gui/MainCont.java:1]: And here is where I become lost.

Comment (TryExceptElse) [inline src/main/java/io/github/tryexceptelse/jdex/fe/gui/MainCont.java:1]: When you have some time again, skype me, I'll walk you through my set up.

**Comment by TryExceptElse** [comment]
Updating skeleton to what should be very close to its final form

**Pull request merged**